src/pages/docs/chat/rooms/media-sharing.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "Sharing media with Ably Chat"
```

Comment:
I think this should be "Media sharing" to match with the nav and the format of other pages?

Reply:
done, thanks